fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -56,6 +56,15 @@ default List<String> listDbNames() {
         return Lists.newArrayList();
     }
 
+    /**
+     * List all warehouse names of connector
+     *
+     * @return a list of string containing all database names of connector
+     */
+    default List<String> listWhNames() {
```

Comment:
I don't think we need to implement `listWhNames()` and `createWarehouse()` in connector, computing warehouse has nothing to do with sr connector.

Reply:
we should remove this function